task-sdk-tests/tests/task_sdk_tests/constants.py

```diff
@@ -28,6 +28,8 @@
 
 DOCKER_COMPOSE_HOST_PORT = os.environ.get("HOST_PORT", "localhost:8080")
 TASK_SDK_HOST_PORT = os.environ.get("TASK_SDK_HOST_PORT", "localhost:8080")
-TASK_SDK_API_VERSION = "2025-08-10"
+
+# Keep this in sync with: airflow/sdk/api/datamodels/_generated.py
+TASK_SDK_API_VERSION = "2025-09-23"
```

Comment:
What does the "Task SDK API Version" actually represent? (What does it differ than `1.1.0`)
I think that it shuold be documented.

<img width="854" height="279" alt="Image" src="https://github.com/user-attachments/assets/2b20283a-9cdb-40db-8e69-79d5a6f1fc26" />